docs/errors/passkeys.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Passkey errors
+description: An index of Clerk errors related to passkeys.
+type: reference
+---
+
+# Passkey errors 
```

Comment:
This page looks different than our other error pages.
Please see any of them, such as https://clerk.com/docs/errors/actor-tokens, to see what format it should look like.

This is another reason why I added `error.errors` in the code examples - because it will show the specific Clerk errors

Reply:
Passkey methods return [ClerkRuntimeError](https://github.com/clerk/javascript/blob/main/packages/shared/src/error.ts#L134), so `error.errors` is undefined. There is no `longMessage` or `shortMessage`.  The only properties are `message` and `code`, so each error on the page would look like this:

> ## `passkey_registration_cancelled`
> 
> ```
> {
 >  code: passkey_registration_cancelled
 >  message: The operation either timed out or was not allowed.
> }
```

Which is redundant, so I just put the message.

Reply:
ohhh I see, thank you for explaining that !